js-exercises/array-utils/arrayUtils.js

```diff
@@ -0,0 +1,57 @@
+function forEach(array, callbackFn) {
+  if(!array.length) throw new TypeError('Passed argument is not an array');
```

Comment:
#1. If an empty array is passed as first argument 'array', the error message would still say "Passed argument is not an array". Because if array is empty, then `array.length === 0` is `false`, hence `!array.length = true`. The error message is not generic in a positive manner.
```suggestion
 if(!array.length) throw new Error('Passed argument is either not an array or is an empty array');
```
- Line 19 and line 36 have same issue.